assets/stylesheets/common/discourse-post-event.scss

```diff
@@ -1,15 +1,14 @@
 $interested: #fb985d;
+$show-interested: inherit;
 
 .discourse-post-event {
   display: flex;
+  max-width: 35rem;
```

Comment:
what is 35 rem?

Reply:
Its just to decrease the width of the container. I saw it used elsewhere and thought it was a nice width to stick to.